### PR TITLE
Publicize Group Commands PrevField and NextField

### PR DESCRIPTION
**Pull request by KevM**
This change makes it possible to create a custom field and support navigation between fields of the group like the built in huh fields do.

## Context 

I am trying to create my own custom input fields and ran into the private `prevField` and `nextField` commands preventing me from behaving like your built-in fields do.

### CODE REVIEW

**Comment by ardnew** [comment]
This should fix #108

**Comment by KevM** [comment]
Fixed the merge conflict with main

**Comment by WTIGER001** [comment]
I am interested in this too.

**Comment by KevM** [comment]
@bashbunni Howdy! ⛵ This the issue I mentioned in Discord. Happy to do what I can to help.

File: .vscode/launch.json
Comment (bashbunni) [inline .vscode/launch.json:1]: Hey, this shouldn't be committed. Mind removing this?

**Review by bashbunni — COMMENTED** [review]
Hey do you mind providing me with an example of how you're hoping to apply these changes? An example of some custom field that requires the `PrevField` and `NextField` functions to be public would be fantastic. I've seen that others are experiencing a similar blocker, but I don't have an example that comes to mind. That would be incredibly helpful.

**Comment by KevM** [comment]
@bashbunni I pushed a new branch to my fork extending this PR [with an example custom field](https://github.com/charmbracelet/huh/commit/953dd4036941630455168595738a3d1a3b5d43c7). I didn't want to add it to this PR as it is pretty buggy and not ready for release. Happy to do that if so desired.

Without this PR it is not possible to move to the next form field by issuing a `huh.NextField` command: [like this](https://github.com/KevM/huh/blob/953dd4036941630455168595738a3d1a3b5d43c7/examples/custom_field/model.go#L165)

**Review by maaslalani — APPROVED** [review]

**Comment by maaslalani** [comment]
@KevM Thank you so much, this makes sense since it is needed to implement custom fields 👍

**Pull request merged**